mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule TestcontainersElixir.MixProject do
   use Mix.Project
 
   @app :testcontainers
-  @version "1.7.0"
+  @version "1.7.1"
```

Comment:
there is a workflow step that verifies if the version is consistent. you need to update https://github.com/testcontainers/testcontainers-elixir/blob/main/lib/util/constants.ex#L5 as well